packages/components/ovh-payment-method/src/ovh-payment-mean.js

```diff
@@ -0,0 +1,294 @@
+
+  const addPaymentMean = (availablePaymentMean, params = {}) => {
+    if (usedRegion === 'US') {
+      return Promise.reject(
+        buildRejectError({
+          status: 403,
+          data: {
+            message: 'addPaymentMean is not available for US world part',
+          },
+        }),
+      );
+    }
+
+    const addParams = params;
+
+    if (has(addParams, 'default')) {
+      addParams.setDefault = addParams.default;
+      delete addParams.default;
+    }
+
+    return usedReketInstance
+      .post(`/me/paymentMean/${availablePaymentMean.meanType}`, addParams)
+      .then((result) => {
+        if (
+          result.url &&
+          availablePaymentMean.meanType !== PAYMENT_MEAN_TYPE_ENUM.bankAccount
+        ) {
+          if (!params.returnUrl) {
+            window.open(result.url, '_blank');
+          } else {
+            window.location = result.url;
+          }
+        }
+
+        return result;
+      });
+  };
+
+  /**
+   *  Edit the given payment mean.
+   *
+   *  @param  {PaymentMean} paymentMean The payment mean to edit
+   *  @return {Promise}
+   */
+  const editPaymentMean = (paymentMean, params) => {
+    if (usedRegion === 'US') {
+      return Promise.reject(
+        buildRejectError({
+          status: 403,
+          data: {
+            message: 'editPaymentMean is not available for US world part',
+          },
+        }),
+      );
+    }
+
+    return usedReketInstance.put(
+      `/me/paymentMean/${paymentMean.meanType}/${paymentMean.id}`,
+      params,
+    );
+  };
+
+  /**
+   *  Set the given payment mean as default payment.
+
+   *  @param  {PaymentMean} paymentMean The paymentMean object to set as default
+   *  @return {Promise}
+   */
+  const setDefaultPaymentMean = (paymentMean) => {
+    return usedReketInstance.post(
+      `/me/paymentMean/${paymentMean.meanType}/${paymentMean.id}/chooseAsDefaultPaymentMean`,
+    );
+  };
+
+  /**
+   *  Delete the given payment mean.
+   *
+   *  @param  {PaymentMean} paymentMean The paymentMean object to delete
+   *  @return {Promise}
+   */
+  const deletePaymentMean = (paymentMean) => {
+    return usedReketInstance.delete(
+      `/me/paymentMean/${paymentMean.meanType}/${paymentMean.id}`,
+    );
+  };
+
+  /* -----  End of Actions on payment means  ------*/
+
+  /**
+   * Get a payment mean with given id and type.
+   *
+   * @param  {Number} paymentMeanId The unique id of the payment mean.
+   * @param  {String} paymentType   The type of the payment mean. (used to build right API url).
+   *
+   * @return {Promise}              Which returns an instance of PaymentMean.
+   */
+  const getPaymentMeanOfType = (paymentMeanId, paymentMeanType, transform) => {
+    if (usedRegion === 'US') {
+      return Promise.reject(
+        buildRejectError({
+          status: 403,
+          data: {
+            message: 'getPaymentMeanOfType is not available for US world part',
+          },
+        }),
+      );
+    }
```

Comment:
this if condition seems to be used in multiple places. You can create a separate method and call it where its required with parameters.

Reply:
Same here, this part will be deprecated ;-)